README.md

```diff
@@ -1,42 +1,24 @@
 # Funding Program Manager
 
-(* FIXME: In one or two paragraphs, describe what the extension does and why one would download it. *)
+This extension forms the basis for application management in CiviCRM: manage funding programs from application to payout.
+The process of submitting and processing funding applications can be centralised:
+Application submission, application review, approval, call for funds, payment instruction and proof of use.
+In order to map the complex process in its entirety, this extension should be supplemented with the following:
```

Comment:
I think we should list all extensions required in the `info.xml` (apart from the Civi core extensions) and call them `requirements` or `dependencies`. To me `should be supplemented` indicates that they are optional. Maybe we can skip them here as they will be part of the [install documentation](https://github.com/systopia/funding/blob/add-documentation/docs/installation.md)...